omnibus/package-scripts/updater-deb/postinst

```diff
@@ -110,16 +96,16 @@ if [ -d "$CONFIG_DIR/runtime-security.d" ]; then
     chown -R root:root ${CONFIG_DIR}/runtime-security.d || true
 fi
 
-$INSTALL_DIR/updater/bin/updater/updater bootstrap -P datadog-agent
+# Set updater helper rights
+chmod 750 ${UPDATER_HELPER}
+setcap cap_setuid+ep ${UPDATER_HELPER}
```

Comment:
follow-up will be on a e2e without cap, in that case we'll fallback to +s

Reply:
Not convinced this is worth TBH, `cap_setuid` has been there for longer than systemd and if a system bans capabilities it would be weird to not also ban +s